core/src/apps/ethereum/tokens.py.mako

```diff
@@ -50,7 +50,7 @@ def token_by_chain_address(chain_id: int, address: bytes) -> EthereumTokenInfo |
 
 
 def _token_iterator(chain_id: int) -> Iterator[tuple[bytes, str, int, str]]:
-% for token_chain_id, tokens in group_tokens(supported_on("trezor2", erc20)).items():
+% for token_chain_id, tokens in group_tokens(supported_on("t2t1", erc20)).items():
```

Comment:
dtto here

Reply:
Done in [1512420](https://github.com/trezor/trezor-firmware/pull/3240/commits/1512420557f66045b63527b6b3f0393024a5b889)